File: samples/oneclickbusiness/app/src/main/java/com/operatortokenocb/network/GetBearerBody.java
package com.operatortokenocb.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class GetBearerBody {

    @SerializedName("accessToken")
    @Expose
    private String accessToken;
    @SerializedName("bundleId")
    @Expose
    private String bundleId;
    @SerializedName("clientId")
    @Expose
    private String clientId;
    @SerializedName("packageName")
    @Expose
    private String packageName;

    public GetBearerBody(String accessToken, String bundleId, String clientId, String packageName) {
        this.accessToken = accessToken;
        this.bundleId = bundleId;
        this.clientId = clientId;
        this.packageName = packageName;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getBundleId() {
        return bundleId;
    }

    public void setBundleId(String bundleId) {
        this.bundleId = bundleId;
    }

    public String getClientId() {
        return clientId;
    }

    public void setClientId(String clientId) {
        this.clientId = clientId;
    }

    public String getPackageName() {
        return packageName;
    }

    public void setPackageName(String packageName) {
        this.packageName = packageName;
    }
}

File: SmartCredentials_aOS/identityprovider/src/main/java/de/telekom/identityprovider/rest/GetBearerBody.kt
/*
 * Copyright (c) 2019 Telekom Deutschland AG
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package de.telekom.identityprovider.rest

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by teodorionut.ganga@endava.com at 23/02/2023
 */
class GetBearerBody(
    @Expose @SerializedName("accessToken") var accessToken: String,
    @Expose @SerializedName("clientId") var clientId: String,
    @Expose @SerializedName("bundleId") var bundleId: String?,
    @Expose @SerializedName("packageName") var packageName: String
)